roles/fips/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: 'Install dracut-fips'
+  package: 
+    name: dracut-fips
+    state: present
+
+- name: 'Run dracut to rebuild initramfs'
+  command: dracut --force
+
+- name: 'Edit kernel command-line to include the fips=1 argument'
+  command: grubby --update-kernel=DEFAULT --args=fips=1
```

Comment:
Is it possible to make this idempotent? Will this be persisted when a kernel is updated?

Reply:
@ekohl I've tried on updating the kernel but the idempotency is not persisted. I've tried on setting the current kernel as default using `grubby --set-default <kernal_path>` but this doesn't make sense as updating the kernel and not using it pointless.

Do you have any thoughts of how we can achieve it.